devices/apa102/apa102_test.go

```diff
@@ -698,21 +693,25 @@ func BenchmarkWriteColorful(b *testing.B) {
 	_, _ = d.Write(pixels[:])
 	b.ResetTimer()
 	for i := 0; i < b.N; i++ {
+		d.Intensity = uint8(i)
```

Comment:
I'm confused as to why it was this way, I probably messed up a prior refactoring, I don't recall. Thanks for catching!

Reply:
I didn't change this either .. the diff presentation is not awesome :)